include/onnx-mlir/Runtime/OMTensor.h

```diff
@@ -86,10 +90,13 @@ OMTensor *omTensorCreate(
  * \brief Create an OMTensor with specified data pointer, shape, rank and
  * element type, manually setting data ptr ownership.
  *
- * Using this constructor, users can
- * specify whether OMTensor owns the data, which subsequently determines whether
- * the memory space underlying the data will be freed or not when OMTensor gets
- * destroyed.
+ * Using this constructor, users can specify whether OMTensor owns the data,
+ * which subsequently determines whether the memory space underlying the data
```

Comment:
Better to clarify what the memory space is, pointed by the pointers passed into this function or allocated by this function?

Reply:
Thanks @chentong319, you are raising a good point. Will clarify.